lbrynet/lbrynet_daemon/Downloader.py

```diff
@@ -104,30 +104,32 @@ def make_downloader(args):
                                            self.payment_rate_manager,
                                            download_directory=self.download_directory,
                                            file_name=self.file_name)
+        def setup_key_fee(wallet_balance):
+            if 'fee' in self.stream_info:
+                self.fee = FeeValidator(self.stream_info['fee'])
+                max_key_fee = self._convert_max_fee()
+                converted_fee = self.exchange_rate_manager.to_lbc(self.fee).amount
+                if converted_fee > wallet_balance:
+                    log.warning("Insufficient funds to download lbry://{}, need {}, have {}".
+                                format(self.resolved_name, converted_fee, wallet_balance))
+                    return defer.fail(InsufficientFundsError())
+                if converted_fee > max_key_fee:
```

Comment:
I'd reverse the order of the two fee checks. max key fee seems more substantial than insufficient wallet balance.